### ISSUE TITLE
Permanent’s LKI and other time related data must be persisted (read only)

### ISSUE THREAD

**Issue by JayDi85** [magefree/mage#12506]
See details in #12482

All time related data like LKI must be persisted to static data in the moment of the save. Current code uses many dynamic methods with game param, and saved permanent return “current” object state instead really saved state. It was broken few years ago.

Possible solution: introduce Permanent:copyToSave method and return permanent object with locked (read only) characteristics.

**Comment by Grath** [comment]
Any chance that some of this might interact with how #8775 is bugged? It'd be fantastic if that could get fixed.

**Comment by JayDi85** [comment]
@Grath nope, I don't see any related code to Maarika, Brutal Gladiator. But I found some ideas about possible bug, see related post in https://github.com/magefree/mage/issues/8775#issuecomment-2184152245.
